### DISCUSSION TITLE
lazer problem

### DISCUSSION THREAD

**Discussion by Zenicior** [ppy/osu#25132] [Q&A]
Hello,

So I was transferring data and files from my ssd onto m.2 and my osu lazer transfered as well. I still have files on the other disc but my osu lazer maps didn't transferand  I can't find them anywhere. I know that beatmaps and scores doesn't store in the file form but is there any way that I can recover my beatmaps back. I can normally transfer beatmaps from stable but I had like 4000 beatmaps exclusively on lazer and I hope that there is a way that I can get them back. 

I also wanted to ask if there is any way that you can get the scores played on lazer back?

Sorry and Thanks.

**Comment by peppy** [comment]
Open osu!, find "change folder location" in settings and point it to where you moved the data files to.
